providers/dns/dns_providers.go

```diff
@@ -78,6 +79,8 @@ func NewDNSChallengeProviderByName(name string) (acme.ChallengeProvider, error)
 		provider, err = gcloud.NewDNSProvider()
 	case "godaddy":
 		provider, err = godaddy.NewDNSProvider()
+	case "he":
```

Comment:
I know that HE may be a common way to refer to the company, but can we use something a little more verbose for this string _and_ the package name? Short identifiers are often used as variable names. How about `hurricanedns`.

Reply:
Thank for your suggestion, it makes sense. If we go that route, should we also rename the package name etc?

The CI failed because library `goquery` in missing in dep. I am not very familiar with dep (and this is my first commit for golang project too) Would you please tell me how to fix that?

Reply:
>  If we go that route, should we also rename the package name etc?

Yes, call the package by the same name.

> The CI failed because library goquery in missing in dep. I am not very familiar with dep (and this is my first commit for golang project too) Would you please tell me how to fix that?

/cc @ldez @wyattjoh

Reply:
@johnnie502 
- Install [dep](https://github.com/golang/dep)
- then do `dep ensure -v`

Reply:
could you also rename the package to `hurricanedns`

Reply:
@idez will do